src/commands/logout/functions.ts

```diff
@@ -32,12 +30,7 @@ export default class Login extends Command {
     await this.stateAggregator.tokens.write();
 
     cli.action.stop();
-    if (flags.json) {
-      cli.styledJSON({
-        status: 0,
-        result: [],
-        warnings: [],
-      });
-    }
+
+    return [];
```

Comment:
I won't hold up this PR for this but you should return _something_.
For these types of commands we often return the env that was logged out.